decidim-core/app/cells/decidim/footer_topics_cell.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "cell/partial"
+
+module Decidim
+  # Topics and their corresponding pages are mapped out to the footer
+  # of the decidim application via cell("decidim/footer_topics", nil).
+  #
+  #
```

Comment:
The format for the Example before was consistent with what we do in other cells. Check out: 

- decidim-participatory_processes/app/cells/decidim/participatory_process_groups/process_filters_cell.rb
- decidim-meetings/app/cells/decidim/meetings/public_participants_list_cell.rb
- decidim-meetings/app/cells/decidim/meetings/attending_organizations_list_cell.rb
- decidim-participatory_processes/app/cells/decidim/participatory_processes/filtered_participatory_processes_cell.rb
- decidim-core/app/cells/decidim/footer_pages_cell.rb
- decidim-core/app/cells/decidim/images_panel_cell.rb
- decidim-core/app/cells/decidim/documents_panel_cell.rb
- decidim-core/app/cells/decidim/endorsers_list_cell.rb
- decidim-core/app/cells/decidim/linked_resources_for_cell.rb
- decidim-core/app/cells/decidim/coauthorships_cell.rb
- decidim-core/app/cells/decidim/progress_bar_cell.rb

```suggestion
  # of the layout.
  #
  #  Example:
  #
  #   cell("decidim/footer_topics", nil)
  #
```